src/components/general/Stepper/__stories__/Stepper.stories.tsx

```diff
@@ -48,7 +48,9 @@ const InteractiveStory = () => {
             <Step title="Select workspace" stepKey="step1">
                 <Item>
                     <div>Press the button to enable navigation to next step.</div>
-                    <Button onClick={() => setProgress([...progress, 1])}>Do work</Button>
+                    <div>
```

Comment:
Why the wrapping div?

Reply:
Wrapping component is flexbox without wrapping div button will stretch to 100% horizontal space.

Reply:
Ahh